components/AltNativeContainer.js

```diff
@@ -7,7 +7,6 @@ var React = require('react-native')
 var mixinContainer = require('./mixinContainer')
 var assign = require('object-assign')
 
-var View = React.View
```

Comment:
Oh I see what's going on. This is broken :( we need to pass in View into this.altRender and React.DOM.div for the other one.